worker/main.py

```diff
@@ -32,6 +32,14 @@
     parser.add_argument('--max-work-dir-size', type=str, default='10g',
                         help='Maximum size of the temporary bundle data '
                              '(e.g., 3, 3k, 3m, 3g, 3t).')
+    parser.add_argument('--max-images-size', type=str, default='10g',
+                        help='Maximum amount of disk space to use to store '
+                             'Docker images (e.g., 3, 3k, 3m, 3g, 3t). '
+                             'The --remove-stale-images flag must be specified '
+                             'in order to enforce this limit.')
+    parser.add_argument('--remove-stale-images', action='store_true',
+                        help='Whether to remove Docker images that have not '
+                             'been used recently when the disk is almost full.')
```

Comment:
Change to clearer wording (--remove-stale-images):

`Whether to keep track of Docker images used by CodaLab and to remove those images that have not been used recently when the disk is almost full. Also see --max-images-size.`